src/Z3/Base.hs

```diff
@@ -2295,6 +2297,22 @@ getDatatypeSortConstructorAccessors c dtSort =
 
 -- TODO: Z3_get_relation_column
 
+mkAtMost :: Context -> [AST] -> Int -> IO AST
+mkAtMost _ctx [] _n = error "Z3.Base.mkAtMost: empty list of expressions"
+mkAtMost  ctx es  n =
+  marshal z3_mk_atmost ctx $ \f ->
+  marshalArrayLen es $ \esLen esArr ->
+  h2c n $ \n' ->
+  f esLen esArr n'
+
+mkAtLeast :: Context -> [AST] -> Int -> IO AST
+mkAtLeast _ctx [] _n = error "Z3.Base.mkAtLeast: empty list of expressions"
+mkAtLeast  ctx es  n =
+  marshal z3_mk_atleast ctx $ \f ->
+  marshalArrayLen es $ \esLen esArr ->
+  h2c n $ \n' ->
+  f esLen esArr n'
+
 -- TODO: Z3_func_decl_to_ast
```

Comment:
@stevana Maybe you could insert the TODOs for `mk_pble`, `mk_pbge`, `mk_pbeq` too, so we have this section in order and up to date again?

https://z3prover.github.io/api/html/group__capi.html

Reply:
Fixed!